pymongo/database.py

```diff
@@ -833,6 +833,114 @@ def command(
                 **kwargs,
             )
 
+    @_csot.apply
```

Comment:
The existing _try_next method simply waits for data to become available or the cursor to be killed. I added a try_next() wrapper around it for testing and it caused this test to hang. What behavior do we expect from the try_next() method?

Reply:
I beleive that's because of the missing cursorType handling. For now let's add it in the unified test runner like I describe here: https://github.com/mongodb/mongo-python-driver/pull/1263#discussion_r1244405528